source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -185,6 +185,10 @@ namespace AdaptiveSharedNamespace
                                                         {"#FFFFD700", "#B2FFD700"}, // warning
                                                         {"#FF8B0000", "#B28B0000"}  // attention
                                                     }};
+        ContainerStyleDefinition goodPalette;
+        ContainerStyleDefinition attentionPalette;
+        ContainerStyleDefinition warningPalette;
+        ContainerStyleDefinition accentPalette;
```

Comment:
Do we want default values for these?

Reply:
As of now, the default value is the same as the predefined `defaultPalette` (NOT the one defined by the user in the imported HostConfig). @matthidinger @andrewleader , thoughts on proper behavior or default values for this?

Reply:
We have a default value for emphasisPalette, correct? I think we should similarly specify default values for goodPalette/etc

Reply:
Sure. What do you want them to be? In the tests, I have them set to the subtle color for each ContainerStyle text.